### PR TITLE
[TACHYON-1543] Update - Show permission info in UI

### PR DESCRIPTION
**Pull request by sundapeng**
Jira: https://tachyon.atlassian.net/browse/TACHYON-1543

The pull request is to show permission info in UI 

@apc999 , @jsimsa , @yupeng9 , @dongche

Thanks!

### CODE REVIEW

File: common/src/main/java/tachyon/util/FormatUtils.java [hunk 1]
```diff
@@ -166,5 +167,23 @@ public static long parseSpaceSize(String spaceSize) {
...
   }
 
+  /**
+   * Formats file permission to human-readable version.
+   *
+   * @param permission file permission
+   * @param isFolder if the Path is folder
+   * @return human-readable version of permission
+   */
+  public static String formatPermission(short permission, boolean isFolder) {
```
Comment (yupeng9) [inline common/src/main/java/tachyon/util/FormatUtils.java]: There's CommandUtils.formatPermission. Could you consolidate both?

Comment (yupeng9) [inline common/src/main/java/tachyon/util/FormatUtils.java]: sorry, just saw your change on `CommandUtils.formatPermission`

Comment (sundapeng) [inline common/src/main/java/tachyon/util/FormatUtils.java]: Got it, I think moving it to `FormatUtils` is better, `UI` calls the method from `CommandUtils` may be a little strange.

Comment (apc999) [inline common/src/main/java/tachyon/util/FormatUtils.java]: SGTM

**Comment by apc999** [comment]
+1 LGTM. Thanks @sundapeng

File: common/src/main/java/tachyon/util/FormatUtils.java [hunk 2]
```diff
@@ -166,5 +167,23 @@ public static long parseSpaceSize(String spaceSize) {
     }
   }
 
+  /**
+   * Formats file permission to human-readable version.
+   *
+   * @param permission file permission
+   * @param isFolder if the Path is folder
```
Comment (jsimsa) [inline common/src/main/java/tachyon/util/FormatUtils.java]: s/isFolder/isDirectory ... s/Path/path

**Comment by jsimsa** [comment]
@sundapeng thanks for the PR ... just one small nit, otherwise it LGTM

**Comment by sundapeng** [comment]
Thank @jsimsa for your review, patch updated.

**Comment by jsimsa** [comment]
thanks ... one more comment, should be a quick fix

**Comment by sundapeng** [comment]
Sorry for missing it, patch updated.

**Comment by jsimsa** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**